app/src/main/java/com/mad41/tripreminder/MainScreen.java

```diff
@@ -0,0 +1,253 @@
+            Toast.makeText(getApplication().getBaseContext(), notes.get(i), Toast.LENGTH_SHORT).show();
+        }
+
+        context = this;
+        //we need the toolbar and drawer to show the menu button
+        toolbar = findViewById(R.id.toolbar);
+        setSupportActionBar(toolbar);
+        drawer = findViewById(R.id.drawer_layout);
+
+        //this will show the menu button
+        ActionBarDrawerToggle toggle = new ActionBarDrawerToggle(this, drawer, toolbar, R.string.navigation_drawer_open, R.string.navigation_drawer_close);
+        drawer.addDrawerListener(toggle);
+        toggle.syncState();
+
+        drawerMenu = findViewById(R.id.drawerMenu);
+        setListener();
+
+        if (savedInstanceState == null) {
+            mgr = getSupportFragmentManager();
+            trns = mgr.beginTransaction();
+            frag1 = new OnGoingFrag();
+            trns.replace(R.id.dynamicFrag, frag1);
+            trns.commit();
+            drawerMenu.setCheckedItem(R.id.btnOngoing);
+        }
+        tripViewModel = new ViewModelProvider(this, ViewModelProvider.AndroidViewModelFactory.getInstance(getApplication())).get(TripViewModel.class);
+
+
+    }
+
+
+    @Override
+    public void startAddTripFragment(Bundle bundle) {
+        mgr = getSupportFragmentManager();
+        FragmentTransaction fragmentTransaction = mgr.beginTransaction();
+
+        fragment.setArguments(bundle);
+        fragmentTransaction.replace(R.id.dynamicFrag, fragment);
+        fragmentTransaction.addToBackStack(null);
+        fragmentTransaction.commit();
+    }
+
+
+    private void setListener() {
+        drawerMenu.setNavigationItemSelectedListener(new NavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                switch (item.getItemId()) {
+
+                    case R.id.btnOngoing:
+                        //if I used mgr from above it will crash if I rotated and then changed fragment, and if I used trns it will crash anyway because it's outside the listener
+                        getSupportFragmentManager().beginTransaction().replace(R.id.dynamicFrag, new OnGoingFrag()).commit();
+                        break;
+                    case R.id.btnHistory:
+                        getSupportFragmentManager().beginTransaction().replace(R.id.dynamicFrag, frag2).commit();
+                        break;
+                    case R.id.btnLanguage:
+                        //LiveData<List<Trip>> trips= MyRoomDataBase.getUserDataBaseInstance(getApplicationContext()).tripDao().getAllTrips();
+                        tripViewModel.getAllNotes().observe(MainScreen.this, new Observer<List<Trip>>() {
+                            @Override
+                            public void onChanged(List<Trip> trips) {
+                                System.out.println(trips.get(0));
+                                WriteHandler.WriteInfireBase(trips);
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.